server/bin/fxa-content-server.js

```diff
@@ -86,6 +86,13 @@ function makeApp() {
     type: 'text/plain'
   }));
 
+  // chrome sends 'application/csp-report' and firefox sends 'application/json'
+  // correct is 'application/csp-report': https://w3c.github.io/webappsec/specs/content-security-policy/
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=1192840
+  app.use(bodyParser.json({
+    type: ['json', 'application/csp-report']
```

Comment:
If chrome sends `application/csp-report`, will express automatically interpret that as JSON data? If not, we'll have to manually decode the data [before this line](https://github.com/mozilla/fxa-content-server/pull/3179/files#diff-98dfa4b189a594a486bd20019c2cdff7R45)